toc/working-groups/service-management.md

```diff
@@ -0,0 +1,95 @@
+# Service Management: Working Group Charter
+
+## Mission
+
+Provides interfaces for service lifecycle within application platforms and adapters to common external service providers.
+
+## Goals
+
+- Define the service extension API for cloud foundry brokered services
+- Provide a flexible adapater layer to common hyperscaler service providers
+- Mantain a set of reference volume service brokers and drivers for CF applications to mount stateful data
+
+## Scope
+- Lead OSBAPI
+- Develop and maintain Cloud Service Brokers for AWS, Azure, and GCP
+- Maintain volume service adapters for NFS and SMB.
+- Develop and maintain ServiceFabrik, a generic BOSH-based and Docker-container-based service instance manager
+
+## Non-Goals
+
+
+
+
+## Proposed Membership
+
+- Technical Lead(s): Marcela Campo
+- Execution Lead(s): Marcela Campo
+
+## Approvers by Area
+### OSBAPI
+* [@Samse](https://github.com/Samse)
+* [@rsampaio](https://github.com/rsampaio)
+
+### Cloud Service Broker
+* [@blgm](https://github.com/blgm)
+* [@FelisiaM](https://github.com/FelisiaM)
+* [@pivotal-marcela-campo](https://github.com/pivotal-marcela-campo) 
+
+
+### Volume Service Adapters
+
+* [@dlresende](https://github.com/dlresende)
+* [@fejnartal](https://github.com/fejnartal)
+* [@totherme](https://github.com/totherme)
+
+### ServiceFabrik
+
+* [@saud89](https://github.com/saud89)
+* [@anoopjb](https://github.com/anoopjb)
+* [@Pooja-08](https://github.com/Pooja-08) 
+* [@swati1102](https://github.com/swati1102)
```

Comment:
I think github will automatically deference `@usernames` so I we don't need to double link here
```suggestion
* @Samse
* @rsampaio

### Cloud Service Broker
* @blgm
* @FelisiaM
* @pivotal-marcela-campo 

### Volume Service Adapters

* @dlresende
* @fejnartal
* @totherme

### ServiceFabrik

* @saud89
* @anoopjb
* @Pooja-08
* @swati1102
```

Reply:
It doesn't link usernames automatically in how it renders Markdown docs, unfortunately.